state/state.go

```diff
@@ -27,12 +27,14 @@ var (
 // but leaves the Consensus.App version blank.
 // The Consensus.App version will be set during the Handshake, once
 // we hear from the app what protocol version it is running.
```

Comment:
The Go doc for this function looks incorrect. Proposal to update it or delete it entirely. Sorry I can't provide a GitHub suggestion because these lines weren't modified.